conformity/fields/country.py

```diff
@@ -27,17 +31,13 @@ def __init__(self, code_filter=lambda x: True, **kwargs):
 
     def errors(self, value):
         if not isinstance(value, six.text_type):
-            return [Error("Not a unicode string")]
+            return [Error('Not a unicode string')]
 
         if value not in self.values:
-            return [Error(
-                "Not a valid country code",
-                code=ERROR_CODE_INVALID,
-                pointer="value",
-            )]
+            return [Error('Not a valid country code')]
```

Comment:
We can completely get rid of the `errors` method entirely if we add this code to the constructor _**after**_ the call to `super`:

```
self._error_message = "Not a valid country code"
```

I would prefer that approach. It simplifies things greatly.

Also, Conformity uses double quotes, not single quotes. I know, this is not consistent with our other projects. I plan to do a commit at some point that coverts to single quotes, but I haven't gotten around to it. Until that point, please use double quotes.

Reply:
I've just changed the constructor to set `_error_message`, and added a call to `super(CountryCodeField, self).errors(value)` inside `errors()`.
I believe that we still need to enforce that the value is a unicode string, and this isn't being done automatically in the `Constant` base class.

Reply:
> I believe that we still need to enforce that the value is a unicode string, and this isn't being done automatically in the `Constant` base class.

Good point!